model/atmosphere/atmosphere_final_increment_cubed_sphere.yaml.j2

```diff
@@ -1,9 +1,9 @@
 output:
   state component:
     filetype: cube sphere history
-    filename: ./anl/atminc.%yyyy%mm%dd.%hh%MM%ssz.nc4
+    filename: ./anl/cubed_sphere_grid_atminc.%yyyy%mm%dd.%hh%MM%ssz.nc4
     provider: ufs
-    fields to write: [ugrd,vgrd,tmp,pressfc,spfh,icmr,clwmr,o3mr]
+    fields to write: {{analysis_variables}}
```

Comment:
```suggestion
    fields to write: {{atmosphere_analysis_variables}}
```

Reply:
All keys have to begin with the model component. This is to ensure that later on if there was some strong coupling there wouldn't be a duplicate key between, say, ocean and atmosphere.

Reply:
@danholdaway Note though that ```analysis_variables``` is taken from the ```jcb-base.yaml.j2``` for the atmosphere in GDASApp from about 5 months ago. I'll need to change that in the GDASApp PR and update it anywhere else it's used for JCB.

Reply:
Although it appears it's not used in many places, so nevermind, should be easy

Reply:
Ok thanks. That might be legacy before this check was enforced. If there's a key in a jcb base that doesn't have the format <component>_key then it likely isn't actually being used in jcb-gdas, otherwise the tests wouldn't be passing.